src/pages/Home/index.tsx

```diff
@@ -51,6 +51,14 @@ const Home = () => {
         icon="waving.svg"

```

Comment:
- **Runtime error check**  
The `AboutContent` object is used without checking if it exists. This could lead to a runtime error if `AboutContent` is null or undefined.
- **Optimization**  
No optimization points found.
- **Security issue**  
No security issues found.